podcasts/AudioUtils.swift

```diff
@@ -1,34 +1,116 @@
 import AVFoundation
 import Foundation
+import Accelerate
+import PocketCastsUtils
 
 class AudioUtils {
     private static let bufferLength = UInt32(Constants.Audio.defaultFrameSize)
     private static let bufferByteSize = Float32(MemoryLayout<Float32>.size)
 
     class func fadeAudio(_ audio: BufferedAudio, fadeOut: Bool, channelCount: UInt32) {
+        if FeatureFlag.accelerateEffects.enabled {
```

Comment:
I don't think this is a problem (at least it wasn't in my tests): changing from one method to the other doesn't matter because the result will be the same right?

I'm just thinking about an occasion in which we switch the feature flag while someone is playing. Based on my real device tests, this doesn't matter, but I want to double-check with you.